plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/core/connection/IOpenShiftConnection.java

```diff
@@ -38,6 +38,8 @@
 	
 	<T extends IResource> List<T> getResources(String kind, String namespace);
 
+	<T extends IResource> T getResource(IResource resource);
```

Comment:
The rationale is that the connection is now used as IOpenShiftConnection and getResource(IResource) was used (in WatchManager) so it was added in the interface definition but I agree I should have copied the Javadoc at least